atlasdb-client/src/main/java/com/palantir/atlasdb/schema/generated/CompactMetadataTable.java

```diff
@@ -513,7 +513,7 @@ public void delete(Iterable<CompactMetadataRow> rows) {
     }
 
     private static Multimap<CompactMetadataRow, CompactMetadataNamedColumnValue<?>> getRowMapFromRowResults(Collection<RowResult<byte[]>> rowResults) {
-        Multimap<CompactMetadataRow, CompactMetadataNamedColumnValue<?>> rowMap = HashMultimap.create();
+        Multimap<CompactMetadataRow, CompactMetadataNamedColumnValue<?>> rowMap = ArrayListMultimap.create();
```

Comment:
was this intentional? This seems a bit random, and I don't know how it got here. Because of how the values are generated I think this is fine, though in general this is a behavioural change

Reply:
Yeah this is from https://github.com/palantir/atlasdb/pull/5855